changes/453.feature.rst

```diff
@@ -0,0 +1 @@
+With "interleaved" syntax, the order of method arguments are now strictly checked.
```

Comment:
This should be a `.removal` change note, as it is a backwards incompatible change (`.removal` will be included the "backwards incompatible" section of the release notes).
```suggestion
The order of keyword arguments used when invoking methods must now match the order they are defined in the Objective-C API. Previously arguments could be in any order.
```